src/common/components/highlight/Highlight.tsx

```diff
@@ -5,25 +5,38 @@ import styles from './highlight.module.scss';
 
 interface HighlightProps {
   className?: string;
+  headingLevel?: number;
   icon: React.ReactNode;
   text: string | React.ReactNode;
   title: string;
 }
 
 const Highlight: React.FC<HighlightProps> = ({
   className,
+  headingLevel = 2,
   icon,
   text,
   title,
 }) => {
+  const titleParts = title.split('\n');
+
   return (
-    <div className={classNames(styles.highlight, className)}>
+    <section className={classNames(styles.highlight, className)}>
       <div className={styles.iconWrapper} aria-hidden={true}>
         {icon}
       </div>
-      <div className={styles.title}>{title}</div>
+      <div
```

Comment:
I assume the heading elements (h1, h2, etc) are not used here directly because of the default styling they have? If so, this can be marked as resolved.